Formula/libbitcoin-explorer.rb

```diff
@@ -16,39 +16,15 @@ class LibbitcoinExplorer < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "libbitcoin"
+  depends_on "libbitcoin-client"
   depends_on "libbitcoin-network"
   depends_on "zeromq"
 
-  resource "libbitcoin-protocol" do
-    url "https://github.com/libbitcoin/libbitcoin-protocol/archive/v3.5.0.tar.gz"
-    sha256 "9deac6908489e2d59fb9f89c895c49b00e01902d5fdb661f67d4dbe45b22af76"
-  end
-
-  resource "libbitcoin-client" do
-    url "https://github.com/libbitcoin/libbitcoin-client/archive/v3.5.0.tar.gz"
-    sha256 "bafa26647f334ecad04fc4bbef507a1954d7e0682f07bd38b90ab66dba5fe0d2"
-  end
-
   def install
     ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin"].opt_libexec/"lib/pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin-client"].opt_libexec/"lib/pkgconfig"
```

Comment:
This gets added automatically if it's a dependency so the line is not needed

Reply:
I thought so too, but if I don't include it I get

```
Package 'libbitcoin-protocol', required by 'libbitcoin-client', not found
```

Reply:
Yes you're right. I didn't see the `libexec`